src/ldc/intrinsics.di

```diff
@@ -679,3 +679,31 @@ static if (LLVM_atleast!6)
 pragma(LDC_intrinsic, "llvm.sideeffect")
     void llvm_sideeffect();
 }
+
+//
+// WEBASSEMBLY-SPECIFIC INTRINSICS
+//
```

Comment:
Thx. - You could use a `version (WebAssembly)` guard, which would serve as documentation too.